pkg/options/caching.go

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
wondering if this file can be moved to `pkg/cloud/services/powervs/cache` or some other relevant place